docs/reference/modules/node.asciidoc

```diff
@@ -89,19 +88,25 @@ be elected to become the master node by the <<modules-discovery,master election
 process>>.
 
 IMPORTANT: Master nodes must have access to the `data/` directory (just like
-`data` nodes) as this is where the cluster state is persisted between node restarts.
+`data` nodes) as this is where the cluster state is persisted between node 
+restarts.
 
-Indexing and searching your data is CPU-, memory-, and I/O-intensive work
-which can put pressure on a node's resources. To ensure that your master
-node is stable and not under pressure, it is a good idea in a bigger
-cluster to split the roles between dedicated master-eligible nodes and
-dedicated data nodes.
+[float]
+[[dedicated-master-node]]
+==== Dedicated master-eligible node
+
+Indexing and searching your data is CPU-, memory-, and I/O-intensive work which 
+can put pressure on a node's resources. To ensure that your master node is 
+stable and not under pressure, it is a good idea to split the roles between 
+dedicated master-eligible nodes and dedicated data nodes. 
```

Comment:
How about something like this?

> It is important for the health of the cluster that the elected master node has the resources it needs to fulfil its responsibilities. If the elected master node is overloaded with other tasks then the cluster may not operate well. In particular, indexing and searching your data can be very resource-intensive, so in large or high-throughput clusters it is a good idea to avoid using the master-eligible nodes for tasks such as indexing and searching. You can do this by configuring three of your nodes to be _dedicated master-eligible nodes_. Dedicated master-eligible nodes only have the `master` role, allowing them to focus on managing the cluster.

Reply:
@DaveCTurner Thank you for your suggestion, I changed the text accordingly.